### PR TITLE
Analytics logging wcs persisting coverageid

### PR DESCRIPTION
**Pull request by cmangeat**
relates to https://github.com/georchestra/georchestra/issues/2049

### CODE REVIEW

**Comment by pmauduit** [comment]
to other reviewers (if any): look at each commit instead of the whole diff in one go

File: ogc-server-statistics/src/main/java/org/georchestra/ogcservstatistics/log4j/OGCServiceParser.java [hunk 1]
```diff
@@ -59,10 +59,16 @@
...
-		{ 	SERVICE_KEYWORD+WFS,SERVICE_KEYWORD+WCS, SERVICE_KEYWORD+WMS, 
-			SERVICE_KEYWORD+ "WMTS", 
-			SERVICE_KEYWORD+"\"WFS\"", SERVICE_KEYWORD+"\"WCS\"", SERVICE_KEYWORD+"\"WMTS\"" };
+	private static final String WMTS = "WMTS";
+	private static final String[] SERVICE_TYPES = {
+		SERVICE_KEYWORD + WFS,
+		SERVICE_KEYWORD + WCS,
+		SERVICE_KEYWORD + WMS,
+		SERVICE_KEYWORD + WMTS,
+		SERVICE_KEYWORD + "\"" + WFS + "\"",
```
Comment (fvanderbiest) [inline ogc-server-statistics/src/main/java/org/georchestra/ogcservstatistics/log4j/OGCServiceParser.java]: I'm really wondering if the lines like ```SERVICE_KEYWORD + "\"" + WFS + "\"",``` are useful

Comment (fvanderbiest) [reply ogc-server-statistics/src/main/java/org/georchestra/ogcservstatistics/log4j/OGCServiceParser.java]: I would be in favor of removing them

**Comment by fvanderbiest** [comment]
Adding a test to show that WCS coverageid is persisted would be useful too.

**Comment by fvanderbiest** [comment]
And, since it is bugfix -> targeting 18.06 branch

**Pull request merged**